### PR TITLE
Updated annotation to attribute

### PR DESCRIPTION
**Pull request by sasha767**
Updated composer. Removed sensio/framework-extra-bundle. Updated annotation to attribute

### CODE REVIEW

**Comment by mcfedr** [comment]
@nonanerz please review

File: composer.json [hunk 1]
```diff
@@ -20,8 +20,7 @@
     "require": {
         "php": ">=8.0",
         "ext-redis": "*",
-        "symfony/framework-bundle": "^5.0|^6.0",
-        "sensio/framework-extra-bundle": "^5.0|^6.0"
+        "symfony/framework-bundle": "^5.0|^6.0"
```
Comment (nonanerz) [inline composer.json:23]: It seems that php-cs-fixer v2 can be removed.

File: tests/bootstrap.php [hunk 2]
```diff
@@ -1,11 +1,8 @@
 <?php
 
 use Composer\Autoload\ClassLoader;
-use Doctrine\Common\Annotations\AnnotationRegistry;
 
 /** @var ClassLoader $loader */
 $loader = require __DIR__.'/../vendor/autoload.php';
 
-AnnotationRegistry::registerLoader([$loader, 'loadClass']);
```
Comment (nonanerz) [inline tests/bootstrap.php:9]: Looks like this bootstrap does not perform any actions anymore. Just remove it

File: tests/config_test.yml [hunk 3]
```diff
@@ -5,6 +5,10 @@ framework:
         strict_requirements: ~
         utf8: true
     secret: 'fake_secret'
+    http_method_override: true
+    handle_all_throwables: true
```
Comment (nonanerz) [inline tests/config_test.yml:9]: Possible breaking change for symfony < 6.3

**Review by nonanerz — COMMENTED** [review]
Also, take a look at routing_anno.yml. The `ekreative_health_check.type` should be `attribute` instead of `annotation`.

File: tests/config_test.yml [hunk 4]
```diff
@@ -6,7 +6,7 @@ framework:
         utf8: true
     secret: 'fake_secret'
     http_method_override: true
-    handle_all_throwables: true
+    handle_all_throwables: false
```
Comment (nonanerz) [inline tests/config_test.yml:9]: The option "handle_all_throwables" is not defined in previous versions. You will receive an error: "Unrecognized option 'handle_all_throwables' under 'framework'."

**Comment by nonanerz** [comment]
@mcfedr i think it's ready to be merged

**Comment by mcfedr** [comment]
@nonanerz i gave you permissions

**Pull request merged**